playbooks/tests/sync-docker.yaml

```diff
@@ -19,7 +19,7 @@
     - ../../conf/satperf.local.yaml
   vars:
     test_sync_docker_count: 2
-    test_sync_docker_url_template: "https://registry-1.docker.io/"
+    test_sync_docker_url_template: "https://registry-1.docker.io*"
```

Comment:
This `...-template` vars are meant to be expanded in a way that asterisk is replaced by number of repo, but in this Docker test, we only have one URL (one registry => without asterisk) and we change that upstream repo name

Reply:
ack, I have updated the URL. Thank you!